src/cpp/rtps/history/ReaderHistory.cpp

```diff
@@ -224,13 +225,8 @@ void ReaderHistory::updateMaxMinSeqNum()
     }
     else
     {
-        auto minmax = std::minmax_element(m_changes.begin(),
-                        m_changes.end(),
-                        [](CacheChange_t* c1, CacheChange_t* c2){
-                        return c1->sequenceNumber < c2->sequenceNumber;
-                    });
-        mp_minSeqCacheChange = *(minmax.first);
-        mp_maxSeqCacheChange = *(minmax.second);
+        mp_minSeqCacheChange = m_changes.front();
```

Comment:
`mp_minSeqCacheChange` and `mp_maxSeqCacheChange` could be a error development point. Maybe it should be better access directly to `m_changes.front()` and `m_changes.back()`

Reply:
Refactored on 85d6e3e8ad5bfef027e3da40c9957f9fcda3b2a9